README.md

```diff
@@ -170,6 +170,13 @@ Please note that this is the upstream documentation for the metrics-utility proj
 
 As the project grows, more guides and references will be added to the `/docs` folder.
 
+## Version mapping
+
+|metrics-utility version|AAP version|
+|-|-|
+|0.4.1|2.4\*|
+|0.5.0|2.5-20250507|
```

Comment:
Do we need the controller version too? 🤔

Reply:
Might be a good addition, yeah.

Except I still don't know how to find the controller version without guessing a commit around the right date, and looking at files to see if changes are there, so, future enhancement? :)